lib/modules/config.js

```diff
@@ -90,25 +90,34 @@ class Config {
     return this.fileSha;
   }
 
-  async addContributor({ login, contributions, name, avatar_url, profile }) {
+  #findOldContributions(username) {
```

Comment:
what's the `#` sign? Is this valid JavaScript? I've never seen that before

Reply:
> what's the `#` sign? Is this valid JavaScript? I've never seen that before

It's private class field at run time.

[MDN - Private Class Fields](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields)